### ISSUE TITLE
Some notices on new update!

### ISSUE THREAD

**Issue by dracosc** [mrknow/filmkodi#381]
Hi again MrKnow! Great work my friend! We really appreciate all your work and really thanks for everything!
I was playing around with the new update the last hour, and i have made some points!
First, i noticed that Oneclickwatch and the new serieswatch added, are based on a site database file! Very nice idea but it is too much work to keep those databases updated! We can help anytime if you ask for it!
Second notice is that, Oneclickwatch, Serieswatch and pelispedia sources on tv shows, only appear if the name of the show is just one word. For example, they appear in Arrow and  Homeland as i tested, but not in Game  of Thrones and Person of Interest! So that has something to do with the title search of the scrapers! 
Another thing is, we have lost watchseries and WSOnline sources. Why is that?
P.S: You can ask for specific help if you need any at anytime! We are not real python developers but we can share some ideas and knowledge for you!
Thanks!

**Comment by bjgood** [comment]
Hi Mr. Know,
Yes, source WSOnline is not providing any links for tv shows at all. They must have changed something on their website.

Regards,
Bob

**Comment by bjgood** [comment]
Hi Mr. Know,
I found out why source wsonline_tv.py stopped providing any links for TV shows. At line 117, in that line, you changed SD to MQ. If you change MQ to SD wsonline_tv.py works. I discovered it quite by accident. On my Firestick I was running 2016.06.18.1 and when my son went to watch The Adventures of Puss in Boots (only available on WSOnline) it worked. I was surprised to say the least because I knew it didn't work on my raspberry pi which is running the latest version of Specto (2016.07.02.2). I compared files and found that discrepancy.

Hope this helps.

Regards,
Bob
